CHANGELOG.md

```diff
@@ -1,3 +1,22 @@
+## [2.23.0] (Unreleased)
+
+BREAKING CHANGES:
+
+ - Remove `terraform.languageServer.maxNumberOfProblems` ([1062](https://github.com/hashicorp/vscode-terraform/pull/1062))
+
+ENHANCEMENTS:
+
+ - Add Module Providers view refresh button ([1065](https://github.com/hashicorp/vscode-terraform/pull/1065))
+
+INTERNAL:
+
+ - dependabot: Reduce interval for dev dependencies to weekly ([1072](https://github.com/hashicorp/vscode-terraform/pull/1072))
+ - Organize Static Features ([1073](https://github.com/hashicorp/vscode-terraform/pull/1073))
```

Comment:
I'd argue this is a user-visible enhancement, so we'd be down-selling it here under `INTERNAL`? 😄

Reply:
Moving classes around is user-visible? Are you referring to #1082?

Reply:
My bad - I was referring to https://github.com/hashicorp/vscode-terraform/pull/1024 which was already shipped :fades-away: